app/src/common/shared/com/igalia/wolvic/browser/SettingsStore.java

```diff
@@ -107,7 +107,7 @@ SettingsStore getInstance(final @NonNull Context aContext) {
     public final static long FXA_LAST_SYNC_NEVER = 0;
     public final static boolean RESTORE_TABS_ENABLED = true;
     public final static boolean BYPASS_CACHE_ON_RELOAD = false;
-    public final static int DOWNLOADS_STORAGE_DEFAULT = INTERNAL;
+    public final static int DOWNLOADS_STORAGE_DEFAULT = EXTERNAL;
```

Comment:
My main concern is that the external location might not exist. I am not sure if that's the case with modern devices but the internal storage will always be there in any case. So I believe that if we want to temporarily remove the option we should stick to internal by default.